config/puma.rb

```diff
@@ -6,6 +6,7 @@
 #
 threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
 threads threads_count, threads_count
+workers_count = ENV.fetch('WEB_CONCURRENCY') { 1 }
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [Sider](https://sider.review) -->